src/components/LoansByCategory/CategoryRowHover.vue

```diff
@@ -307,7 +308,7 @@ export default {
 			handler(channel) {
 				this.name = _get(channel, 'name');
 				this.url = _get(channel, 'url');
-				this.loans = _get(channel, 'loans.values');
+				this.loans = _filter(_get(channel, 'loans.values'));
```

Comment:
what does this _filter do?

Reply:
Handles cases where `loans` is something like `[null, null]` (I only encountered this in testing)